### PR TITLE
feat(CODEOWNERS): add codeowners per module

### PR DESCRIPTION
**Pull request by theStiggySpd**
This PR does the following:
- Add Codeowners PER modules (existing as of today)

Here are the new/updated teams [https://github.com/orgs/SAP/teams?query=ux-tools](https://github.com/orgs/SAP/teams?query=ux-tools)

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,48 @@
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
-* @ux-tools-team-tools @ux-tools-team-elements
+# Module: store
+/packages/store @SAP/ux-tools-store
+
+# Module: fe-fpm-writer
+/packages/fe-fpm-writer/ @SAP/ux-tools-fpm-writer
```
Comment (Klaus-Keller) [inline .github/CODEOWNERS:7]: To follow the naming convention, shouldn't it be `@SAP/ux-tools-fe-fpm-writer`?
```suggestion
/packages/fe-fpm-writer/ @SAP/ux-tools-fe-fpm-writer
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,3 +1,48 @@
...
+
+# Module: fiori-freestyle-writer
+/packages/fiori-freestyle-writer/ @SAP/ux-tools-freestyle-writer
```
Comment (Klaus-Keller) [inline .github/CODEOWNERS:10]:
```suggestion
/packages/fiori-freestyle-writer/ @SAP/ux-tools-fiori-freestyle-writer
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,3 +1,48 @@
...
+
+# Module: logger
+/packages/logger/ @SAP/ux-tools-logger
+
+# Module: odata-service-writer
+/packages/odata-service-writer/ @SAP/ux-tools-service-writer
```
Comment (Klaus-Keller) [inline .github/CODEOWNERS:16]:
```suggestion
/packages/odata-service-writer/ @SAP/ux-tools-odata-service-writer
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,3 +1,48 @@
...
+
+# Module: ui5-application-writer
+/packages/ui5-application-writer/ @SAP/ux-tools-ui5-app-writer
```
Comment (Klaus-Keller) [inline .github/CODEOWNERS:19]:
```suggestion
/packages/ui5-application-writer/ @SAP/ux-tools-ui5-application-writer
```

**Review by Klaus-Keller — COMMENTED** [review]
Found some inconsistencies in naming

File: .github/CODEOWNERS (hunk 1 above)
Comment (theStiggySpd) [reply .github/CODEOWNERS]: @Klaus-Keller Thanks for the feedback. I have updated the team names as you have suggested.
Initial mismatch was as I was thinking to keeping the names a bit shorter, but I get your point. Lets be consistent. 
https://github.com/SAP/open-ux-tools/commit/de4667c65a5badfb9d1a9212d326df1d989ed782

File: .github/CODEOWNERS (hunk 2 above)
Comment (theStiggySpd) [reply .github/CODEOWNERS]: https://github.com/SAP/open-ux-tools/commit/de4667c65a5badfb9d1a9212d326df1d989ed782

File: .github/CODEOWNERS (hunk 3 above)
Comment (theStiggySpd) [reply .github/CODEOWNERS]: https://github.com/SAP/open-ux-tools/commit/de4667c65a5badfb9d1a9212d326df1d989ed782

File: .github/CODEOWNERS (hunk 4 above)
Comment (theStiggySpd) [reply .github/CODEOWNERS]: https://github.com/SAP/open-ux-tools/commit/de4667c65a5badfb9d1a9212d326df1d989ed782

**Review by Klaus-Keller — APPROVED** [review]
@theStiggySpd, looks consistent now. Just a remark:
should we still have the entry 
```
* @ux-tools-team-tools @ux-tools-team-elements
```
to match all new generated folders/files?

**Comment by theStiggySpd** [comment]
> @theStiggySpd, looks consistent now. Just a remark: should we still have the entry
> 
> ```
> * @ux-tools-team-tools @ux-tools-team-elements
> ```
> 
> to match all new generated folders/files?

Hello @Klaus-Keller 
This needs a bit more discussion and handle in later pr, if thats ok with you.
I want to get this merged sooner now. Will be great to have one more approval from @SAP/ux-tools-admin colleagues

**Review by kranthie-sap — APPROVED** [review]

**Pull request merged**